pennylane/registers.py

```diff
@@ -25,16 +25,16 @@ def registers(register_dict):
     This function helps to group qubits and abstract away the finer details of running quantum
     algorithms. Register names and their total number of wires are typically known in advance,
     but managing the specific wire range for each register can be a challenge. The ``qml.registers()``
-    function creates a dictionary that maps register name to :class:`~.Wires` object. Moreover,
+    function creates a dictionary that maps register names to :class:`~.Wires` objects. Moreover,
     it allows one to input a nested structure where registers contain sub-registers, as illustrated
     in the examples below.
 
     Args:
-        register_dict (dict): a dictionary where keys are register names and values are either
+        register_dict (dict): A dictionary where keys are register names and values are either
```

Comment:
I kind of prefer the original one. See [here](https://docs.pennylane.ai/en/stable/development/guide/documentation.html#punctuation-and-spelling) for example.

Reply:
isn't this a proper sentence though?

Reply:
I think this fits more in the "or longer sentence fragments" but I also don't mind. I have made the changes.